qa/vagrant-helpers.rb

```diff
@@ -34,10 +50,17 @@ def self.parse(lines)
 
     def self.execute(cmd)
       Open3.popen3(cmd) do |stdin, stdout, stderr, wait_thr|
-        { :stdout => stdout.read.chomp, :stderr => stderr.read.chomp,
-          :exit_status => wait_thr.value.exitstatus }
+        ExecuteResponse.new(stdin, stdout.read.chomp, stderr.read.chomp, wait_thr.value.exitstatus)
       end
     end
 
+    def self.execute_successfully(cmd)
```

Comment:
for simplicity, I think we should just have one `execute` command with this behavior